packages/xml-lsp-server/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@xml-tools/xml-lsp-server",
+  "description": "XML language server",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "repository": {
```

Comment:
a main property is needed to expose APIs to consumers

Reply:
not sure what did you mean, I've added: "main": "src/server.ts"